providers/frame/common.ts

```diff
@@ -1,8 +1,7 @@
-import { File } from "../../codegen/frame/mod.ts"
 import { ss58, testUser } from "../../crypto/mod.ts"
 import * as $ from "../../deps/scale.ts"
 
-const DEFAULT_TEST_USER_COUNT = 100_000
+const DEFAULT_TEST_USER_COUNT = 1_000
```

Comment:
#741

Reply:
Interesting, locally I didn't noticed a meaningful difference.

What do you mean by pre-compute?

Was the blocking related to this `for` loop?
https://github.com/paritytech/capi/blob/da4d7a94036b2da54e0ba981d70df01379195fbb/providers/frame/common.ts#L17-L22

Reply:
Yes, that for-loop was taking about 7 seconds on my machine, and it seems about 30 on ci.

Reply:
> What do you mean by pre-compute?

All of the test-users are deterministic, so we should be able to take the 100_000 test user addresses, compute them, and store them in a file somewhere, rather than computing them every time.

Reply:
perfect!